debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/CaseSensitivenessIT.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.sqlserver;
+
+import java.sql.SQLException;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.fest.assertions.Assertions;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.debezium.config.Configuration;
+import io.debezium.connector.sqlserver.SqlServerConnectorConfig.SnapshotMode;
+import io.debezium.connector.sqlserver.util.TestHelper;
+import io.debezium.embedded.AbstractConnectorTest;
+import io.debezium.util.Testing;
+
+/**
+ * Integration test to verify behaviour of detabase with and without case sensistive names.
+ *
+ * @author Jiri Pechanec
+ */
+public class CaseSensitivenessIT extends AbstractConnectorTest {
+
+    private SqlServerConnection connection;
+
+    @Before
+    public void before() throws SQLException {
+        TestHelper.createTestDatabase();
+        connection = TestHelper.testConnection();
+
+        initializeConnectorTestFramework();
+        Testing.Files.delete(TestHelper.DB_HISTORY_PATH);
+    }
+
+    @After
+    public void after() throws SQLException {
+        if (connection != null) {
+            connection.close();
+        }
+    }
+
+    @Test
+    public void caseInsensitiveDatabase() throws Exception {
+        connection.execute(
+                "CREATE TABLE MyTableOne (Id int primary key, ColA varchar(30))",
+                "INSERT INTO MyTableOne VALUES(1, 'a')"
+        );
+        TestHelper.enableTableCdc(connection, "MyTableOne");
+        testDatabase();
+    }
+
+    @Test
+    public void caseSensitiveDatabase() throws Exception {
```

Comment:
How does this test the originally reported issue?

Reply:
The original issue is verified by this change - https://github.com/debezium/debezium-incubator/pull/64/files#diff-b306088505d9239e03c8adb3da8488deR27

Reply:
Uff, ok, that's subtle :)